ETSMobile/ETSMobile/Util/Request.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Request.swift
+//  ETSMobile
+//
+//  Created by Felix Leveille on 2018-10-02.
+//  Copyright © 2018 ApplETS. All rights reserved.
+//
+
+import Foundation
+
+class Request {
```

Comment:
There is some overlap with the `MonETSClient` class, not sure it will be needed in the end. Let me know what you guys think.